lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -91,7 +91,7 @@ class ChatInfoView extends StatelessWidget {
             const SizedBox(height: 8),
             if (c.isMonolog) ...[
               _avatar(c, context),
-              const NotesBlock.info(),
+              _actionsMonolog(c, context),
```

Comment:
`NotesBlock.info()` isn't used anymore. Or perhaps should be refactor it to contain both avatar and information in it? And it can be used on `Chat` page as well with its `info` flag set to `false` or `true`.